.github/workflows/srtool.yml

```diff
@@ -9,10 +21,15 @@ jobs:
       matrix:
         chain: ["statemine", "westmint"]
     steps:
+      - name: Get Time
+        id: time
+        uses: nanzm/get-time-action@v1.1
```

Comment:
It's weird to me that we'd need to rely on a third party action to just get the time. As a general rule I like to only use third party actions where necessary. Maybe this is necessary though.

Reply:
We actually don't and I agree. I guess we may use that more so a homebrewed action maybe better. For now, I think I can remove it entirely.

Reply:
I removed it